epictrack-api/src/api/schemas/work_v2.py

```diff
@@ -7,13 +7,25 @@
 from api.schemas.eao_team import EAOTeamSchema
 from api.schemas.federal_involvement import FederalInvolvementSchema
 from api.schemas.ministry import MinistrySchema
-from api.schemas.phase import PhaseSchema
 from api.schemas.project import ProjectSchema
 from api.schemas.staff import StaffSchema
 from api.schemas.substitution_act import SubstitutionActSchema
 from api.schemas.work_type import WorkTypeSchema
 
 
+class WorkPhaseSchema(
+    AutoSchemaBase
+):  # pylint: disable=too-many-ancestors,too-few-public-methods
+    """Work phase model schema class"""
+
+    class Meta(AutoSchemaBase.Meta):
+        """Meta information"""
```

Comment:
Why do you have to name it work_v2.py, we could name it work_phase. Plus I always wanted to bring those schemas sitting outside respone or request folder to inside

Reply:
This is the old schema and I think we are not using this schema anymore. File was probably changed when I did a search for current_phase_id in the entire code base and changed all occurrence to current_work_phase_id